examples/ShoppingAds/AddShoppingSmartAd.php

```diff
@@ -348,13 +348,10 @@ private static function addSmartShoppingAdGroupAd(
         int $customerId,
         string $adGroupResourceName
     ) {
-        // Creates a new Smart Shopping ad.
-        $ad = new Ad(['shopping_smart_ad' => new ShoppingSmartAdInfo()]);
-
         // Creates a new ad group ad and sets the Smart Shopping ad to it.
         $adGroupAd = new AdGroupAd([
             // Sets the ad to the ad created above.
```

Comment:
You need to adjust this comment accordingly.

Reply:
Done.